.pre-commit-config.yaml

```diff
@@ -1,13 +1,9 @@
 repos:
--   repo: https://github.com/psf/black
-    rev: 23.7.0
-    hooks:
-    - id: black
-      language_version: python3.10
-
--   repo: https://github.com/pycqa/isort
-    rev: 5.12.0
-    hooks:
-    - id: isort
-      name: isort (python)
-      language_version: python3.10
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  # Ruff version.
+  rev: v0.1.5
```

Comment:
If the ruff.tolm requires greater-than-0.2 I assume this needs the 0.2.1 in https://github.com/sot/skare3/commit/6b1cb925b3057fa135fd8473c0d6e65e8af0cc9f but probably doesn't matter for the PR.

Reply:
Fixed